src/main.rs

```diff
@@ -44,5 +45,11 @@ async fn run_sync() -> anyhow::Result<()> {
 		.finish();
 	tracing::subscriber::set_global_default(subscriber)
 		.context("Setting default tracing subscriber failed")?;
-	sync_ldap_users_to_zitadel(config).await
+
+	// Run the chosen SYNC
```

Comment:
Sure, the endpoint & LDAP are not mutually in conflict per se. So it can be also something like (instead of the `else` condition):
```
if config.deactivate_from_endpoint() {
   sync_deactivate_emails(config).await
}
sync_ldap_users_to_zitadel(config).await
```

Depends how we want to approach this. Will we be calling this crate once and let it do everything at the same time. Or will we be calling this crate on multiple occasions targeting different outcomes? For example, once per day to delete emails from endpoint, other two times per day to sync ldap to zitadel, and so on.

How often do we actually do the sync currently?